mmv1/products/iap/WebRegionBackendService.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'WebRegionBackendService'
+base_url: 'projects/{{project}}/iap_web/compute-{{region}}/services/{{name}}'
+self_link: 'projects/{{project}}/iap_web/compute-{{region}}/services/{{name}}'
+# This resource is only used to generate IAM resources. They do not correspond to real
+# GCP resources, and should not be used to generate anything other than IAM support.
+exclude_resource: true
+description: |
+  Only used to generate IAM resources
+exclude_validator: true
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  method_name_separator: ':'
+  parent_resource_type: 'google_compute_region_backend_service'
+  parent_resource_attribute: 'web_region_backend_service'
+  fetch_iam_policy_verb: :POST
+  allowed_iam_role: 'roles/iap.httpsResourceAccessor'
+  iam_conditions_request_type: :REQUEST_BODY
+id_format: 'projects/{{project}}/iap_web/compute-{{region}}/services/{{name}}'
+import_format: ['projects/{{project}}/iap_web/compute-{{region}}/services/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_backend_service_basic'
+    primary_resource_id: 'default'
+    vars:
+      backend_service_name: 'region_backend-service'
```

Comment:
Based on the configuration in use, I think this should be region_backend_service_name: https://github.com/GoogleCloudPlatform/magic-modules/blob/c9d82af3ffdbaf21d436632a3bf62dc0aed5f86e/mmv1/templates/terraform/examples/region_backend_service_basic.tf.erb#L2